app/assets/stylesheets/locals/plain.css.scss

```diff
@@ -10,5 +10,11 @@
             display: none;
             // Title at top of transcript isn't necessary: We know what we're looking at.
         }
+        .play-from-here::before {
+            content: '\25B6'; // right-pointing triangle
+            cursor: pointer;
+            color: $dark-blue;
+            padding-right: 0.3em;
+        }
```

Comment:
@mccalluc injecting content via CSS sets off a red flag for me. But I guess in this case we're using content (characters) as style (arrows), so it's a valid exception to the rule.

merging